lib/iris/experimental/stratify.py

```diff
@@ -82,31 +82,33 @@ def relevel(cube, src_levels, tgt_levels, axis=None, interpolator=None):
     that are generally monotonic in the direction of interpolation, such as
     height/pressure or salinity/depth.
 
-    Parameters
-    ----------
-    cube : :class:`~iris.cube.Cube`
+    Args:
+
+    * cube (`iris.cube.Cube`):
```

Comment:
@tv3141 why have the intersphinx references been removed throughout this change? Are the references not desired, or am I missing something?

Reply:
@dkillick By changing the style of the docstring a line became too long and caused a Stickler error. I could not find a way to break a bullet point without violating rst syntax, and thus shortened the line by removing the references.
I will look into using `numpydoc`, it should fix the original issue without any other changes.